README.md

```diff
@@ -1,15 +1,27 @@
-# Interactive Message Adapter for Node and Express
+# Slack Interactive Messages for Node
 
 [![Build Status](https://travis-ci.org/slackapi/node-slack-interactive-messages.svg?branch=master)](https://travis-ci.org/slackapi/node-slack-interactive-messages)
 [![codecov](https://codecov.io/gh/slackapi/node-slack-interactive-messages/branch/master/graph/badge.svg)](https://codecov.io/gh/slackapi/node-slack-interactive-messages)
 
-The adapter provides a simplified API to route message actions and options in your app. It handles
-common tasks and best practices so that you don't need to.
+Build your Slack Apps with rich and engaging user interactions using buttons, menus, and dialogs.
+The packages makes it easy to start with sensible and secure defaults.
+
+The adapter gives you a meaningful API to handle actions from all of Slack's interactive
+message components ([buttons](https://api.slack.com/docs/message-buttons),
+[menus](https://api.slack.com/docs/message-menus), and [dialogs](https://api.slack.com/dialogs)). It
+can be used to start its own HTTP server, or can be plugged into an existing server as an
+[Express](http://expressjs.com/) middleware.
```

Comment:
"It can be used..." -> "Use it as a independent HTTP server or plug it into an existing server as Express middleware."    I'd put this on its own line too, important enough info to eyeball.